docs/source/configuration/metrics.mdx

```diff
@@ -56,11 +56,38 @@ apollo_router_http_request_duration_seconds_bucket{le="0.9"} 1
 
 > Note that if you haven't run a query against the router yet, you'll see a blank page because no metrics have been generated!
 
-### Available metrics
+## Datadog via OTLP
+
+To use Datadog, you must configure the Datadog agent to accept OTLP metrics. This can be done by adding the following to your `datadog.yaml`:
+
+```yaml title="datadog.yaml"
+otlp_config:
+  receiver:
+    protocols:
+      grpc:
+        endpoint: 127.0.0.1:4317
```

Comment:
This will be confusing to anyone trying to configure their router to run in a containerised (k8s/docker) environment. More typically, in k8s, the endpoint will be something like:
```
# Datadog agent collector over grpc
            endpoint: "${env.NODE_IP}:4317"
```
Should we document that first and then document the `localhost` configuration second and explain that's ok for development or running standalone machine?

Reply:
I think that you're probably right, but we should apply this across the board for the docs. We should be targeting k8s first in all our router documentation.
@shorgi WDYT?

Reply:
Agree that Docker and k8s are likelier than localhost, though will need to review rest of docs for any instances localhost is intended dev env. 

And a general strategy for specifying user-specified variables docs is to call them out as `<user-provided-variable>`. In this case for the endpoint, `endpoint: <dd-node-ip>:4317`

Reply:
OK, I've used env expansion where appropriate and where env expansion is not possible I've used the format that Edward has specified.


aa7cd48